### DISCUSSION TITLE
Conditionally hide and show Steps in a Stepper

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#10251] [CLI - PWA mode] [answered]
Hello Friends,

I have implemented a q-stepper but i want to hide and show certain step conditionally. for example i have step A B and C
if  am on step A and i want to go to B but B is false it should not appear the stepper it should take me to C.

i have a sample code here 
https://codepen.io/akashramtohul4/pen/jOmxvBN?editors=1010

can someone please help

**Comment by ontwikkelfabriek** [comment, marked as answer]
@akashramtohul4 

Here's  a codepen:
https://codepen.io/ontwikkelfabriek/pen/xxdJPEM

**Reply by akashramtohul4** [reply]
Thanks 👍
